### PR TITLE
[hip-hcc] build & run tests, add missing deps

### PR DESCRIPTION
**Pull request by s-bernard**
Add a check step to build and run tests. Unfortunatly, to do that,
we should first install HIP in current directory. Then rerun cmake to
use correct INSTALL_PREFIX.

HIP actually depends on HCC and rocminfo to work.

Finally, remove flag BUILD_TYPE because Release is by default.

### CODE REVIEW

**Comment by s-bernard** [comment]
Note: I can’t make the test pass

**Comment by s-bernard** [comment]
Note: the tests pass if I compile it the same way in an ubuntu docker with official HCC package.

**Comment by s-bernard** [comment]
Links to #138

File: hip-hcc/PKGBUILD [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 pkgver=3.3.0
-pkgrel=5
+pkgrel=6
 pkgdesc="Heterogeneous Interface for Portability ROCm"
 arch=('x86_64')
 url="https://github.com/ROCm-Developer-Tools/HIP"
 license=('MIT')
-depends=('hsa-rocr')
+depends=('hsa-rocr' 'hcc' 'rocminfo')
 makedepends=('libelf' 'cmake' 'python' 'hcc' 'comgr' 'git')
```
Comment (ranisalt) [inline hip-hcc/PKGBUILD:10]: Please remove `hcc` from `makedepends` as it's already a runtime `depends`

File: hip-hcc/.SRCINFO [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 	makedepends = python
-	makedepends = hcc
 	makedepends = comgr
 	makedepends = git
 	depends = hsa-rocr
+	depends = hcc
+	depends = rocminfo
 	provides = hip
 	conflicts = hip
+	options = !strip
```
Comment (ranisalt) [inline hip-hcc/.SRCINFO:18]: Is this really needed? I usually refrain from using !strip, especially when building from source.

Comment (s-bernard) [reply hip-hcc/.SRCINFO:18]: Actually yes, it is mandatory to pass the reduction test as describe in #135. You may check yourself to be sure that it is not happening only on my system.

**Comment by acxz** [comment]
Closing this issue since hip-hcc is deprecated

**Pull request closed without merging**